frontend/src/views/account-mgmt/ManageFacilityView.vue

```diff
@@ -34,22 +34,24 @@
     <v-row>
       <v-col cols="12" class="ml-6 pr-9 pt-0">
         <v-card elevation="0" variant="outlined" class="w-100">
-          <v-skeleton-loader v-if="licences?.length > 0" :loading="loading" type="table-tbody">
-            <v-expansion-panels v-model="panel" multiple>
+          <v-skeleton-loader :loading="loading" type="table-tbody">
+            <v-expansion-panels v-if="licences?.length > 0" v-model="panel" multiple>
               <v-expansion-panel v-for="licence in licences" :key="licence.licenceId" :value="licence.licenceId">
                 <v-expansion-panel-title class="header-label">
                   <LicenceHeader :licence="licence" />
                   <v-col cols="auto">
-                    <v-icon icon="fa:fa-regular fa-pen-to-square" class="" @click.stop="addEditLicense(licence.licenceId)"></v-icon>
+                    <AppButton variant="text" :disabled="loading">
+                      <v-icon icon="fa:fa-regular fa-pen-to-square" class="transaction-icon" @click.stop="addEditLicense(licence.licenceId)"></v-icon>
+                    </AppButton>
                   </v-col>
                 </v-expansion-panel-title>
                 <v-expansion-panel-text>
                   <LicenceDetails :licence="licence" />
                 </v-expansion-panel-text>
               </v-expansion-panel>
             </v-expansion-panels>
+            <div v-else class="pa-5">0 Licences</div>
           </v-skeleton-loader>
-          <div v-else class="pa-5">0 Licences</div>
         </v-card>
       </v-col>
     </v-row>
```

Comment:
I believe the code on line 76 of ManageFacilityView.vue is redundant.

Reply:
Not sure why redundant... i moved that line into the skel loader and the v-if from skel loader to v-expansion-panels because previously when screen was loading the licences section showed Licences: 0 during loading.